src/rpc/service_eth.go

```diff
@@ -135,18 +146,18 @@ func (service *EthService) SendRawTransaction(ctx context.Context, s string) (*c
 		Signer: newSigner.Signer,
 	}
 
-	opts.Value = tx.Cost()
+	opts.Value = new(big.Int).Sub(tx.Cost(), tx.Value())
```

Comment:
```suggestion
	opts.Value = big.NewInt(0).Sub(tx.Cost(), tx.Value())
```

I don't know, if there is any penalty on `new(big.Int)` vs `big.NewInt(0)`, but this is how [examples](https://pkg.go.dev/math/big) initialize it.